### ISSUE TITLE
[BUG] On Windows HyperV dynamic memory is on allowing more than the default 8G

### ISSUE THREAD

**Issue by jeffmaury** [code-ready/crc#1069]
### General information

  * OS: Windows
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
# Put `crc version` output here
crc version: 1.6.0+8ef676f
OpenShift version: 4.3.0 (embedded in binary)
```
  
## CRC status
```bash
# Put `crc status` output here
CRC VM:          Stopped                       
OpenShift:       Stopped                       
Disk Usage:      0B of 0B (Inside the CRC VM)  
Cache Usage:     53.03GB                       
Cache Directory: C:\Users\JeffMAURY\.crc\cache 
```

## CRC config
```bash
# Put `crc config view` output here
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
Nom de l’hôte:                              DESKTOP-U2EVGBM                                                                                 
Nom du système d’exploitation:              Microsoft Windows 10 Professionnel                                                              
Version du système:                         10.0.18363 N/A version 18363                                                                    
Fabricant du système d’exploitation:        Microsoft Corporation                                                                           
Configuration du système d’exploitation:    Station de travail autonome                                                                     
Type de version du système d’exploitation:  Multiprocessor Free                                                                             
Propriétaire enregistré:                    Jeff MAURY                                                                                      
Organisation enregistrée:                                                                                                                   
Identificateur de produit:                  00330-80819-37372-AA630                                                                         
Date d’installation originale:              18/11/2019, 14:37:22                                                                            
Heure de démarrage du système:              01/03/2020, 12:48:54                                                                            
Fabricant du système:                       LENOVO                                                                                          
Modèle du système:                          20LAS3NJ0H                                                                                      
Type du système:                            x64-based PC                                                                                    
Processeur(s):                              1 processeur(s) installé(s).                                                                    
                                            [01] : Intel64 Family 6 Model 142 Stepping 10 GenuineIntel ~1910 MHz                            
Version du BIOS:                            LENOVO N27ET36W (1.22 ), 04/07/2019                                                             
Répertoire Windows:                         C:\WINDOWS                                                                                      
Répertoire système:                         C:\WINDOWS\system32                                                                             
Périphérique d’amorçage:                    \Device\HarddiskVolume1                                                                         
Option régionale du système:                fr;Français (France)                                                                            
Paramètres régionaux d’entrée:              fr;Français (France)                                                                            
Fuseau horaire:                             (UTC+01:00) Bruxelles, Copenhague, Madrid, Paris                                                
Mémoire physique totale:                    32 379 Mo                                                                                       
Mémoire physique disponible:                7 150 Mo                                                                                        
Mémoire virtuelle : taille maximale:        63 099 Mo                                                                                       
Mémoire virtuelle : disponible:             16 133 Mo                                                                                       
Mémoire virtuelle : en cours d’utilisation: 46 966 Mo                                                                                       
Emplacements des fichiers d’échange:        C:\pagefile.sys                                                                                 
Domaine:                                    WORKGROUP                                                                                       
Serveur d’ouverture de session:             \\DESKTOP-U2EVGBM                                                                               
Correctif(s):                               8 Corrections installées.                                                                       
                                            [01]: KB4534132                                                                                 
                                            [02]: KB4516115                                                                                 
                                            [03]: KB4517245                                                                                 
                                            [04]: KB4524569                                                                                 
                                            [05]: KB4528759                                                                                 
                                            [06]: KB4537759                                                                                 
                                            [07]: KB4538674                                                                                 
                                            [08]: KB4532693                                                                                 
Carte(s) réseau:                            5 carte(s) réseau installée(s).                                                                 
                                            [01]: Intel(R) Ethernet Connection (4) I219-LM                                                  
                                                  Nom de la connexion : Ethernet                                                            
                                                  État :                Support déconnecté                                                  
                                            [02]: TAP-Windows Adapter V9                                                                    
                                                  Nom de la connexion : Ethernet 2                                                          
                                                  DHCP activé :         Oui                                                                 
                                                  Serveur DHCP :        10.36.119.254                                                       
                                                  Adresse(s) IP                                                                             
                                                  [01]: 10.36.116.204                                                                       
                                                  [02]: fe80::d527:58a7:4965:c2ad                                                           
                                            [03]: Hyper-V Virtual Ethernet Adapter                                                          
                                                  Nom de la connexion : vEthernet (Default Switch)                                          
                                                  DHCP activé :         Non                                                                 
                                                  Adresse(s) IP                                                                             
                                                  [01]: 192.168.93.17                                                                       
                                                  [02]: fe80::e09c:3931:f493:1205                                                           
                                            [04]: Intel(R) Dual Band Wireless-AC 8265                                                       
                                                  Nom de la connexion : Wi-Fi                                                               
                                                  DHCP activé :         Oui                                                                 
                                                  Serveur DHCP :        192.168.0.254                                                       
                                                  Adresse(s) IP                                                                             
                                                  [01]: 192.168.0.42                                                                        
                                                  [02]: fe80::8548:85c0:593:b8e9                                                            
                                                  [03]: 2a01:e35:2ffe:3020:7481:189b:ced:9911                                               
                                                  [04]: 2a01:e35:2ffe:3020:7471:15eb:5f03:9137                                              
                                                  [05]: 2a01:e35:2ffe:3020:6d8e:fea0:d856:45e6                                              
                                                  [06]: 2a01:e35:2ffe:3020:2db2:d31:e571:866e                                               
                                                  [07]: 2a01:e35:2ffe:3020:10ef:d61d:2dda:ab2e                                              
                                                  [08]: 2a01:e35:2ffe:3020:8548:85c0:593:b8e9                                               
                                            [05]: Bluetooth Device (Personal Area Network)                                                  
                                                  Nom de la connexion : Connexion réseau Bluetooth                                          
                                                  État :                Support déconnecté                                                  
Configuration requise pour Hyper-V:         Un hyperviseur a été détecté. Les fonctionnalités nécessaires à Hyper-V ne seront pas affichées.
```

### Steps to reproduce

  1. crc start
  2. 
  3. 
  4. 

### Expected

Memory in use should not exceed the default 8G

### Actual

After a while the HyperV manager will display more than 8G

### Logs

You can start crc with `crc start --log-level debug` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
Duplicate of #607

**Comment by gbraad** [comment]
https://github.com/code-ready/crc/issues/607#issuecomment-532741155

Dynamic Memory is also the default for Hyper-V and for Minishift. Hyper-V can never assign more than is available to the system, so this is a convience to be hands-off. Does it cause issues to the stability of OpenShift?

**Comment by jeffmaury** [comment]
No but it prevents me from starting another VM when CRC is started.

**Comment by gbraad** [comment]
This can be disabled on a per-VM basis.
![image](https://user-images.githubusercontent.com/1894/76284191-8b226800-62d7-11ea-9467-31ad22fcc8f9.png)

or use:
```powershell
PS> Set-VMMemory crc -DynamicMemoryEnabled $false
```

Note: for follow-ups please use the original issue
